### PR TITLE
chore: ensure website is only released from tags on master branch

### PR DESCRIPTION
**Pull request by okdistribute**

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
           WIN_CSC_LINK: ${{ secrets.WIN_CSC_LINK }}
           WIN_CSC_KEY_PASSWORD: ${{ secrets.WIN_CSC_KEY_PASSWORD }}
       - name: Update website release
-        if: contains(github.ref, '/tags/')
+        if: contains(github.ref, '/master/') && contains(github.ref, '/tags/')
```
Comment (gmaclennan) [inline .github/workflows/nodejs.yml:56]: This won't work unfortunately. When this is triggered by a tag, the ref will be the tag, not the branch (see https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context). I'm not entirely sure how to solve this.

Comment (okdistribute) [reply .github/workflows/nodejs.yml:56]: Hmm, we could then check if we're on the master branch inside the script, maybe? I'll look at this next week.

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -57,7 +57,10 @@ jobs:
           WIN_CSC_LINK: ${{ secrets.WIN_CSC_LINK }}
           WIN_CSC_KEY_PASSWORD: ${{ secrets.WIN_CSC_KEY_PASSWORD }}
       - name: Update website release
-        if: contains(github.ref, '/tags/')
+        if: contains(github.ref, '/tags/') && github.event_name === 'push'
+        on:
```
Comment (okdistribute) [inline .github/workflows/nodejs.yml:61]: not sure if 'on' works at this level

**Comment by okdistribute** [comment]
Had a shower thought that this could be in it's own workflow, which would allow `on.branch.master` and `contains(github.ref, /tag/)`

**Comment by okdistribute** [comment]
We will instead release to the website manually

**Pull request closed without merging**